ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/ConfigCommitter.scala

```diff
@@ -158,15 +158,15 @@ private[kvutils] class ConfigCommitter(
 
     ctx.set(
       configDedupKey(ctx.getParticipantId, result.submission.getSubmissionId),
-      DamlStateValue.newBuilder.build
-    )
-
-    StepStop(
-      DamlLogEntry.newBuilder
-        .setRecordTime(buildTimestamp(ctx.getRecordTime))
-        .setConfigurationEntry(configurationEntry)
+      DamlStateValue.newBuilder
+        .setSubmissionDedup(DamlSubmissionDedupValue.newBuilder)
```

Comment:
In a previous PR I accidentally removed generating the empty dedup value. In case of a `oneof` protobuf type we need to have a value set (even if it's empty) -- hence these changes.

Reply:
OK, ideally we should have that in a separate PR but I understand it's not worth the hassle, unless there's an SDK release with this issue though.

Reply:
Yes. protobuf allows setting at most one value in a `oneof` so unless our code tries to read a value the serialization/deserialization should work. However, we could remove these empty dedup messages altogether in a separate PR.